tests/db/dynamodb_test.py

```diff
@@ -46,3 +46,35 @@ def test_query_user():
     assert user == strict_users[0]
 
     ddb.delete_user('abc_123')
+
+
+def test_store_retrieve_team():
```

Comment:
These should be marked as `db` once the other pull request is merged, correct?

Reply:
yes.